pyproject.toml

```diff
@@ -78,6 +78,8 @@ no_implicit_optional = true
 scripts_are_modules = true
 show_error_codes = true
 warn_redundant_casts = true
+allow_untyped_decorators = true
```

Comment:
Let's do this in another PR, if we really want to do this. But let's keep the discussion in that PR.